tripal_chado/src/Plugin/Field/FieldType/ChadoSequenceCoordinatesDefault.php

```diff
@@ -70,11 +70,11 @@ public static function tripalTypes($field_definition) {
 
     // Get the property terms by using the Chado table columns they map to.
     $storage = \Drupal::entityTypeManager()->getStorage('chado_term_mapping');
-    $mapping = $storage->load( 'core_mapping' );
+    $mapping = $storage->load('core_mapping');
 
-    $record_id_term = $mapping->getColumnTermId( 'feature', 'feature_id' );
-    $ft_uniqname_term = $mapping->getColumnTermId( 'feature', 'name' );
+    $ft_uniqname_term = $mapping->getColumnTermId('feature', 'name');
 
+    $feature_id_term = $mapping->getColumnTermId('featureloc', 'feature_id');
```

Comment:
Hood catch! I agree with getting the term for the featureloc.feature_id in case we want to be more specific there :-)